test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/CSharpEndToEndTests.cs

```diff
@@ -181,66 +180,54 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
                 { "location", "Seattle" }
             };
 
-            var headers = new HeaderDictionary();
-            headers.Add("accept", "text/plain");
-
-            HttpRequest request = HttpTestHelpers.CreateHttpRequest("POST", string.Format("http://localhost/api/httptrigger-dynamic"), headers, input.ToString());
-            request.ContentType = "application/json";
-
-            Dictionary<string, object> arguments = new Dictionary<string, object>
+            HttpRequestMessage request = new HttpRequestMessage
             {
-                { "input", request },
-                { ScriptConstants.SystemTriggerParameterName, request }
+                RequestUri = new Uri(string.Format("http://localhost/api/httptrigger-dynamic")),
+                Method = HttpMethod.Post,
+                Content = new StringContent(input.ToString())
             };
-            await Fixture.Host.CallAsync("HttpTrigger-Dynamic", arguments);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
+            request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+
+            HttpResponseMessage response = await Fixture.Host.HttpClient.SendAsync(request);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
 
-            var response = request.HttpContext.Items[ScriptConstants.AzureFunctionsHttpResponseKey];
-            // Assert.Equal(HttpStatusCode.OK, response.);
-               
-            // string body = await response.Content.ReadAsStringAsync();
-            // Assert.Equal("Name: Mathew Charles, Location: Seattle", body);
+            string body = await response.Content.ReadAsStringAsync();
+            Assert.Equal("Name: Mathew Charles, Location: Seattle", body);
         }
 
         [Fact(Skip = "Investigate test failure")]
```

Comment:
The test has been updated... did you mean to remove the `Skip` ?

Reply:
Not this one, I updated it quickly to see if I could get it running with the new pattern. If not, I left it skipped and moved on.